### ISSUE TITLE
Test Failure in test_theanocode.py

### ISSUE THREAD

**Issue by kshitij10496** [sympy/sympy#10208]
While running the tests, I found this [test](https://github.com/sympy/sympy/blob/master/sympy/printing/tests/test_theanocode.py#L177) fails with an `AssertionError`.
Can someone verify the validity of the test failure ?
If verified, I would like to work on this.

Ping @asmeurer @gxyd @leosartaj

**Comment by leosartaj** [comment]
It is failing for me.

```
 ./bin/test sympy/printing/tests/test_theanocode.py
================================================================ test process starts =================================================================
executable:         /home/leosartaj/anaconda/envs/my/bin/python  (2.7.10-final-0) [CPython]
architecture:       64-bit
cache:              yes
ground types:       python 
random seed:        25639858
hash randomization: on (PYTHONHASHSEED=3019924720)

sympy/printing/tests/test_theanocode.py[27] ...................F.......                                                                         [FAIL]

______________________________________________________________________________________________________________________________________________________
______________________________________________ sympy/printing/tests/test_theanocode.py:test_MatrixSlice ______________________________________________
  File "/home/leosartaj/projects/sympy/sympy/printing/tests/test_theanocode.py", line 177, in test_MatrixSlice
    assert tuple(Yt.owner.op.idx_list) == (slice(1,2,3), slice(4,5,6))
AssertionError

================================================ tests finished: 26 passed, 1 failed, in 0.73 seconds ================================================
DO *NOT* COMMIT!
```

But this shouldn't be happening. Anything to do with versions? @asmeurer

**Comment by asmeurer** [comment]
This is fixed at https://github.com/sympy/sympy/pull/10084/files#diff-6bb9fdf4f4680648a0fec6cdc44cde01

**Comment by kshitij10496** [comment]
Thats cool

**Comment by kshitij10496** [comment]
I think this can be closed now

**Comment by StanczakDominik** [comment]
This does seem to have been fixed by #10084. Could probably be closed.
